rust/gui-client/src-common/src/auth.rs

```diff
@@ -205,14 +213,26 @@ impl Auth {
     ///
     /// Performs I/O
     fn get_token_from_disk(&self) -> Result<Option<SessionAndToken>, Error> {
-        let actor_name = match std::fs::read_to_string(actor_name_path()?) {
-            Ok(x) => x,
+        // Read the actor_name file, then let the session file override it if present.
+
+        let mut session = Session::default();
+        match std::fs::read_to_string(actor_name_path()?) {
+            Ok(x) => session.actor_name = x,
             // It can happen with dev systems that actor_name.txt doesn't exist
             // even though the token is in the cred manager.
             // In that case we just say the app is signed out
             Err(e) if e.kind() == std::io::ErrorKind::NotFound => return Ok(None),
-            Err(e) => return Err(Error::ReadActorName(e)),
+            Err(e) => return Err(Error::ReadFile(e)),
         };
+        match std::fs::read_to_string(session_data_path()?) {
+            Ok(x) => {
+                session = serde_json::from_str(&x).map_err(|_| Error::Serde)?;
+                firezone_telemetry::Hub::main()
+                    .configure_scope(|scope| scope.set_tag("account_slug", &session.account_slug));
```

Comment:
Is there any value in setting this as part of the "user" concept that Sentry has? https://docs.rs/sentry/latest/sentry/struct.Scope.html#method.set_user